app/tests/spec/models/user.js

```diff
@@ -142,6 +142,75 @@ function (chai, sinon, p, Constants, Session, FxaClient, User) {
         });
     });
 
+
+    describe('in memory caching of signed in account', function () {
+      it('getSignedInAccount returns same instance from setSignedInAccount', function () {
+        var account = user.initAccount({ uid: 'uid', email: 'email'});
+        return user.setSignedInAccount(account)
+          .then(function () {
+            assert.equal(user.getSignedInAccount(), account);
+          });
+      });
+
+      it('getSignedInAccount returns same instance when called multiple times', function () {
+        return user.setSignedInAccount({ uid: 'uid', email: 'email'})
+          .then(function () {
+            assert.equal(user.getSignedInAccount(), user.getSignedInAccount());
```

Comment:
[`assert.equal` is the same as `==`](http://chaijs.com/api/assert/#equal), We want `assert.strictEqual` here.